web-console/src/lib/functions/deltalake/inferConfig.spec.ts

```diff
@@ -0,0 +1,115 @@
+import { expect, test } from '@playwright/experimental-ct-react'
+
+import { inferDeltaLakeStorageConfig } from './inferConfig'
+
+// http://pkerschbaum.com/blog/using-playwright-to-run-unit-tests
+
+test('Test DeltaLake config inference', async ({}) => {
```

Comment:
we should have more tests for this based on the issues reported by leonid e.g.,:

- json options should not just appear
- timestamp_column shouldn't remain in JSON when they are empty
- test that checks that you can't set both columns snapshotversion/snapshot delta